### ISSUE TITLE
Performance of ResNet-50 initialized randomly on waterbirds dataset

### ISSUE THREAD

**Issue by jiawei151** [izmailovpavel/spurious_feature_learning#2]
Hi @izmailovpavel ! Thanks for your great work and I really enjoy it. In section 6 and figure 3, it was reported that 'random init' base model achieved 0.9 mean group accuracy, however in my experiment, this number is about 0.6 (on testset). I'm wondering whether there are some hyper parameters I should change? Thanks a lot.

**Comment by izmailovpavel** [comment]
Hi @jiawei151, thank you for the message! Note that the test set is group balanced, while the mean accuracy is computed by reweighting the groups by their proportions in the training set. Here's an example:

```python
wb_group_train_sizes = np.array([3498, 184, 56, 1057]).astype(float)
wb_group_train_sizes /= wb_group_train_sizes.sum()

def get_base_mean_acc(accs):
    return (np.array([accs["accuracy_0_0"], accs["accuracy_0_1"], 
                      accs["accuracy_1_0"], accs["accuracy_1_1"]]) * wb_group_train_sizes).sum()
```

**Comment by jiawei151** [comment]
> 

I got it! Thanks a lot for your detailed explanation.
